app/controllers/api/verify/document_capture_controller.rb

```diff
@@ -0,0 +1,81 @@
+module Api
+  module Verify
+    class DocumentCaptureController < BaseController
+      self.required_step = 'document_capture'
+      include ApplicationHelper
+
+      def create
+        result = @form ||= Idv::ApiDocumentVerificationForm.new(
+          verify_params,
+          liveness_checking_enabled: liveness_checking_enabled?,
+          analytics: analytics,
+        ).submit
+
+        if result.success?
```

Comment:
do we want to add any analytics logging here? I feel like in our app, almost any time we submit a form object, we log the result hash to events.log